src/command_modules/azure-cli-resource/azure/cli/command_modules/resource/_params.py

```diff
@@ -25,7 +21,10 @@
 
 register_cli_argument('resource', 'resource_name', resource_name_type)
 register_cli_argument('resource', 'api_version', help='The api version of the resource (omit for latest)', required=False)
-register_cli_argument('resource', 'resource_provider_namespace', help=argparse.SUPPRESS, required=False)
+
+for item in ['tag', 'update', 'show', 'delete', 'exists']:
+    register_cli_argument('resource {}'.format(item), 'resource_provider_namespace', ignore_type)
```

Comment:
>'resource {}' [](start = 26, length = 13)

Why the for loop to go through each item when before, the type was applied to everything under 'resource'?